flutter/lib/src/binding_wrapper.dart

```diff
@@ -114,16 +138,21 @@ mixin SentryWidgetsBindingMixin on WidgetsBinding {
   void handleDrawFrame() {
     super.handleDrawFrame();
 
-    try {
-      final endTimestamp = _clock?.call();
-      if (_startTimestamp != null &&
-          endTimestamp != null &&
-          _startTimestamp!.isBefore(endTimestamp)) {
-        _frameTimingCallback?.call(_startTimestamp!, endTimestamp);
-      }
-    } catch (_) {
-      if (_options.automatedTestMode) {
-        rethrow;
+    if (_isTrackingActive && isFramesTrackingInitialized()) {
+      try {
+        _stopwatch?.stop();
+        if (_stopwatch!.elapsedMilliseconds >
+            _expectedFrameDuration!.inMilliseconds) {
+          final endTimestamp = _clock!.call();
```

Comment:
Same for the clock. If the logic where/when it's initialzed changes, we crash here if it's null.